scheduler/config-k8s.edn

```diff
@@ -46,6 +46,7 @@
            :user-metrics-interval-seconds 60}
  :nrepl {:enabled? true
          :port #config/env-int "COOK_NREPL_PORT"}
+ :pools {:default "gamma"}
```

Comment:
After this change was made, I had problems bringing up cook, so I reverted it. 

Any idea whats going on?

Reply:
Ah, I've been running without pools in the database locally for k8s testing. If you keep it out and skip the seed pools step, it should work.

Reply:
Aaah. Nice.